gradle/java.gradle

```diff
@@ -55,6 +43,9 @@ if (JavaVersion.current().isJava8Compatible()) {
 // add all of the artifacts above to our archives list
 artifacts {
     archives testJar
-    archives sourceJar
-    archives javadocJar
+}
+
+java {
```

Comment:
We moved it from the exported-project.gradle to the java.gradle so that all the subprojects which use java.gradle would get it consistently. There were a couple projects that were getting both sourcesJar and sourceJar before we made an improvement.  It's not just moved within a single file.  Some subprojects apply java.gradle but not exported-project.gradle, but exported-project.gradle applies java.gradle.

Reply:
ah ok, then makes sense to do this.